cmd/ike/cmd/develop.go

```diff
@@ -107,6 +109,7 @@ func NewDevelopCmd() *cobra.Command {
 	developCmd.Flags().Bool("no-build", false, "always skips build")
```

Comment:
why a -build and a -no-build? Why isn't "no-build" just not having a -build?

Reply:
b/c you might have it in the config - still I can try to resolve that case but thought that would be more explicit.

Reply:
Made it hidden 7626fe9

Reply:
ah, ic...